.vscode/launch.json

```diff
@@ -41,6 +41,22 @@
             "outFiles": ["${workspaceFolder}/dist/src/test/**/*.js"],
             "preLaunchTask": "npm: watch"
         },
+        {
+            "name": "Extension Tests (current file)",
+            "type": "extensionHost",
+            "request": "launch",
+            "runtimeExecutable": "${execPath}",
+            "args": [
+                "--extensionDevelopmentPath=${workspaceFolder}",
+                "--extensionTestsPath=${workspaceFolder}/dist/src/test/index.js"
+            ],
+            "env": {
+                "TEST_REPORT_DIR": "${workspaceFolder}/.test-reports",
+                "TEST_FILE": "${relativeFileDirname}/${fileBasenameNoExtension}"
```

Comment:
Just making sure: what happens if the current file isn't a test?

Reply:
The glob matches and runs 0 tests successfully 😄